File: Project/src/app/model/teacherSetting.ts
export class TeacherSet{
    public id:any;
    public title:string = "";
    public description:string = "";
    public phone:string = "";
    public side:string = "";

    constructor(id:any,title:string,description:string,phone:string,side:string){
            this.id=id;
            this.title=title;
            this.description=description;
            this.phone=phone;
            this.side=side;
        }
}

File: WorkAPI/WebAPI3/BindingModel/TeacherSetting.cs
namespace WebAPI3.BindingModel
{
    public class TeacherSetting
    {
        public TeacherSetting(Guid id, string firstName, string lastName, string position, string sex, string email, string photoFile,
            string idTeacher, string title, string description, string phone, string side)
        {
            Id = id;
            PhotoFile = photoFile;
            FirstName = firstName;
            LastName = lastName;
            Email = email;
            Position = position;
            Sex = sex;
            IdTeacher=idTeacher;
            Title = title;
            Description = description;
            Phone = phone;
            Side = side;
        }
        public Guid Id { get; set; }
        public string PhotoFile { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Email { get; set; }

        public string Position { get; set; }

        public string Sex { get; set; }
        public string IdTeacher { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Phone { get; set; }
        public string Side { get; set; }
    }
}
